app/pages/new-dose-medication/new-dose-medication.ts

```diff
@@ -37,8 +41,7 @@ export class NewDoseMedicationPage implements OnInit {
         );
     }
 
-    public onSubmit(): DoseSeries {
-        /* Will need to save via service. */
+    public onSubmit(): Promise<DoseMedication> {
```

Comment:
I thought this form was going to create a new `DoseSeries`, so this change would be incorrect.

It should use a `DoseSeriesService.save` to send the `DoseSeries` to the server, which will create and save the med and schedule information.

Right? @jking31cs ?